t/discovery/dns/mix.t

```diff
@@ -128,3 +128,4 @@ connect to 127.0.0.1:1053
 connect to 127.0.0.1:1053
 connect to 127.0.0.1:1053
 connect to 127.0.0.1:1053
+connect to 127.0.0.1:1053
```

Comment:
Why do you add this?

Reply:
earlier it was trying 4 times but when resolv.conf is passed it tries one more times(5 times)